File: tinyQuest.server/Models/Party.cs
namespace tinyQuest.Models
{
    public class Party{

            public Party()
        {
        }

        public int Id {get; set;}
        public string creatorId {get; set;}
        public int hero1Id {get; set;}
        public int hero2Id {get; set;}
        public int hero3Id {get; set;}
        public int hero4Id {get;set;}
        public Profile Creator {get; set;}

    }
}

File: tinyQuest.client/src/Models/Party.js
export class Party {
  constructor(data = {}) {
    this.id = data.id || data._id
    this.creatorId = data.creatorId
    this.hero1Id = data.hero1Id
    this.hero2Id = data.hero2Id
    this.hero3Id = data.hero3Id
    this.hero4Id = data.hero4Id
  }
}
